app/src/main/java/com/amaze/filemanager/ui/views/appbar/SearchView.java

```diff
@@ -199,7 +203,30 @@ public boolean isShown() {
     return searchViewLayout.isShown();
   }
 
-  public interface SearchListener {
+    private void initSearchViewColor(MainActivity a ) {
+        switch (a.getAppTheme().getSimpleTheme(a)) {
+            case LIGHT:
+                searchViewLayout.setBackgroundResource(R.drawable.search_view_shape);
+                searchViewEditText.setTextColor(Utils.getColor(a, android.R.color.black));
+                clearImageView.setColorFilter(ContextCompat.getColor(a, android.R.color.black),
+                        PorterDuff.Mode.SRC_ATOP);
+                backImageView.setColorFilter(ContextCompat.getColor(a, android.R.color.black),
+                        PorterDuff.Mode.SRC_ATOP);
+                break;
+            case DARK:
+            case BLACK:
```

Comment:
This doesn't seem correct, are you sure dark and black have same background?

Reply:
Hello @EmmanuelMess thanks for the review, I have added the code for the dark mode too. 
Regards